cmd/ipfs/goreq.go

```diff
@@ -0,0 +1,3 @@
+// +build !go1.5		
+
+`IPFS needs to be built with go version 1.5 or greater`
```

Comment:
Yeah, i thought about that, but i'd rather have builds fail than have them succeed and then crash.

Letting the build succeed might lead to shipping bad binaries accidentally